code/modules/admin/DB ban/functions.dm

```diff
@@ -566,4 +572,11 @@ datum/admins/proc/DB_ban_unban_by_id(var/id)
 
 			output += "</table></div>"
 
-	usr << browse(output,"window=lookupbans;size=900x700")
\ No newline at end of file
+	usr << browse(output,"window=lookupbans;size=900x700")
+
+/proc/flag_account_for_forum_sync(var/ckey)

```

Comment:
No leading `var/` in proc arguments, please

Reply:
Changed